src/styles/main.scss

```diff
@@ -1,3 +1,18 @@
+@import "./utils/variables";
+@import "./blocks/calendar.scss";
+
+*,
+*::before,
+*::after,
 body {
   margin: 0;
+  padding: 0;
+
+}
+
+body {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 100vh;
```

Comment:
typically for centering items in body we use `min-height` but that also works fine 😄